extensions/adobe/experience/customerJourneyManagement/email-channel-context.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/email-channel-context",
```

Comment:
Remove the "xdm" from the $id path. Should be https://ns.adobe.com/experience/customerJourneyManagement/email-channel-context

Reply:
@cmathis : I noticed current test scripts do not validate the schemas under the "extension" folder. For validating the schemas with examples in this PR, I have to modify the test scripts locally to validate them. 
While running the schemas validating scripts it gives the error that "id" should match the following regex, and because of that I added the "xdm" in the uri path.  I will remove the "xdm" from the uri.

`(https://ns\.adobe\.com/xdm/[a-z0-9-/]*)|(http://schema\.org/.*)|(http://ns.adobe.com/adobecloud/core/1.0.*)|([http|https]://ns\.adobe\.com/adobecloudplatform/.*)|(https://tools\.ietf\.org/html/draft-kelly-json-hal-08/.*)|(http://www.iptc\.org/.*)|(https://id3\.org/id3v2.4/.*)`


On a side note, I assume this is a known issue, that current validation scripts do not validate schemas under the extension folder.